doc/modules/mod_jingle_sip.md

```diff
@@ -1,26 +1,88 @@
 ### Module Description
 
-This module enabled Jingle to SIP and SIP to Jingle translator.
-When this module is enalbed, MongooseIM will intercept any iq set jingle stanza with action:
+This module enables Jingle to SIP and SIP to Jingle translation.
+When this module is enabled, MongooseIM will intercept any Jingle IQ set stanza with action:
+
 * session-initiate
 * session-terminate
 * session-accept
 * transport-info
 
-and translate it to SIP messages with approprite SDP content based on the details in the Jingle stanza.
+and translate it to SIP messages with appropriate SDP content based on the details in the Jingle stanza.
 
-The translation back from SIP to Jingle is done for following SIP messages:
+The translation back from SIP to Jingle is done for following SIP requests:
 
-* `INVITE` - with additionall callback for following response codes:
-   * `200`
-   * `180` and `183`
-   * `486` when the call's recipient rejects it
-   * from `400` to `600` - other error codes indicating session terminate
-* `re-INVITE` - `INVITE` message sent for established session
+* `INVITE`
+* `re-INVITE` - `INVITE` message sent for an accepted session
 * `CANCEL`
 * `BYE`
 * `INFO`
 
+and following responses to the INVITE request:
+
+* `200` when the call invite was accepted
+* `180` and `183` to indicate that the invitation was sent to the device
+* `486` when the call's recipient rejects it
+* from `400` to `600` - other error codes indicating session termination
+
+#### Jingle to SIP translation
+
+The table below summarises the translation both ways for standard Jingle and SIP messages:
+
+| Jingle action | SIP message | comment |
+| ------------- | ----------- | ------- |
+| `session-initiate` | `INVITE` request | |
+| `session-accept` | `200 OK` response | |
+| `session-terminate` with reason `success` | `BYE` request | Only for accepted session |
+| `session-terminate` with reason `decline` | `CANCEL` request | When sent by call's initiator |
+| `session-terminate` with reason `decline` | `486 Busy Here` response | When sent by the invite user |
+| `transport-info` | `INFO` request | |
+
+##### Ringing notification
+
+Both Jingle and SIP has the `ringing` notification.
+It's generated as a response code `180 Ringing` by a SIP entity when the INVITE is sent to the device.
+In SIP world there is also `183 Session Progress` response code generated in some cases.
```

Comment:
In SIP world a`183 Session Progress` response code is also generated in some cases.